instance/tests/models/test_utils.py

```diff
@@ -851,21 +851,22 @@ def test_purge_no_prefix(self):
 
     def test_purge_with_prefix(self):
```

Comment:
Based on [this comment](https://github.com/open-craft/opencraft/pull/589/files#r437483538), this test can be removed since the `purge()` method will just call the `<consul client>.kv.delete()` method and we will be testing the `consul` client library.